examples/src/test/scala/org/virtuslab/ideprobe/ModuleTest.scala

```diff
@@ -51,8 +77,8 @@ class ModuleTest extends IdeProbeFixture with ScalaPluginExtension with RobotPlu
     val modulesFromConfig = intelliJ.config[Seq[String]]("modules.test")
     val runnerNameFragmentOpt = intelliJ.config.get[String]("runner")
     val moduleRefs = modulesFromConfig.map(ModuleRef(_))
-    val runConfigs = moduleRefs.map(moduleRef => TestRunConfiguration(moduleRef, runnerNameFragmentOpt))
-    runConfigs.map(config => config.module -> intelliJ.probe.run(config)).foreach {
+    val runConfigs = moduleRefs.map(moduleRef => TestRunConfiguration.Module(moduleRef))
```

Comment:
Indeed JUnitRunConfiguration was mapped to concrete run configuration directly, I think now TestScope is more accurate name for this class.

Reply:
IMO the problem is that now TestScope took over the JUnitRunConfiguration (API for JUnit and generic test run became unified). We would have to run JUnitRunConfiguration with the object called TestScope then.

Reply:
it is ok, we run JUnit based on TestScope. If we were to have some custom junit stuff, we can introduce case class JUnitRunConfiguration(scope: TestScope, otherParam: Type)